.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+## Tracking Issue: #*issue id*
+
+
+
+**Note:**
+If you are directly creating a PR then please create an issue for it as well
```

Comment:
Please add these two lines as comments on the PR. You can use HTML comment tag for this `<!--- comment -->`